### DISCUSSION TITLE
Support Windows' context menu key in osu!

### DISCUSSION THREAD

**Discussion by ziv-vy** [ppy/osu#29042] [Ideas] [answered]
### Type

Game behaviour

### Note

This might be the most pointless bug report of all time. The devs might even need to create priority:4 thanks to this.

### Bug description

When pressing this key on your keyboard, the context menu should open. 
![image](https://github.com/user-attachments/assets/c22b0bf8-757b-4c58-9218-65c7ccdea5f9)
(yes this is so underused that I literally have to attach an image of said key)

### Version

2024.718.1

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/16336851/compressed-logs.zip)

**Comment by frenzibyte** [comment]
No need for priority:4, this goes in discussions listing.

**Comment by ziv-vy** [comment]
Since this issue is really niche, I'd suggest to simply rebind the menu key to right click outside of gameplay.

**Reply by bdach** [reply, marked as answer]
no

**Reply by ziv-vy** [reply]
If that's not going to happen, I don't think any other solution are time-efficient.
